atecc608a_se.c

```diff
@@ -153,7 +152,8 @@ psa_status_t atecc608a_deinit()
     return atecc608a_to_psa_error(atcab_release());
 }
 
-static psa_status_t atecc608a_export_public_key(psa_key_slot_number_t key,
+static psa_status_t atecc608a_export_public_key(psa_drv_se_context_t *drv_context,
```

Comment:
Isn't compiler complaining about the unused parameter here and in the other cases? I suggest void casting them, or leaving the argument names out to prevent compiler warnings raising in `-Werror` builds.